src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/nomis/UserServiceTest.kt

```diff
@@ -118,16 +119,16 @@ class UserServiceTest {
     fun `prison users only`() {
       whenever(nomisUserApiService.findUsersByFirstAndLastName("first", "last")).thenReturn(
         listOf(
-          NomisUserSummaryDto("U1", "1", "F1", "l1", false, null, "u1@justice.gov.uk"),
-          NomisUserSummaryDto("U2", "2", "F2", "l2", false, null, null),
-          NomisUserSummaryDto("U3", "3", "F3", "l3", false, PrisonCaseload("MDI", "Moorland"), null)
+          NomisUserSummary("U1", "1", "F1", "l1", false, null, "u1@justice.gov.uk"),
+          NomisUserSummary("U2", "2", "F2", "l2", false, null, null),
+          NomisUserSummary("U3", "3", "F3", "l3", false, PrisonCaseload("MDI", "Moorland"), null)
```

Comment:
Should this now be PrisonUserSummary?  and should we rename the directories to prison rather than nomis - or possibly later (I know we discussed this)